dist/ng-currency.js

```diff
@@ -57,6 +57,19 @@ angular.module('ng-currency', [])
                     }
                 }
 
+                function reformatViewValue(){
+                    var formatters = ngModel.$formatters,
+                        idx = formatters.length;
+
+                    var viewValue = ngModel.$modelValue;
+                    while (idx--) {
+                      viewValue = formatters[idx](viewValue);
+                    }
+
+                    ngModel.$setViewValue(viewValue);
```

Comment:
I'm not sure. This code solves my problem. Maybe you should create a separate issue and PR for that?